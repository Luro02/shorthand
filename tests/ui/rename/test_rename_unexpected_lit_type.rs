use shorthand::ShortHand;

#[derive(ShortHand)]
struct Command {
    #[shorthand(rename(b"binary"))]
    value: String,
}

fn main() {}
