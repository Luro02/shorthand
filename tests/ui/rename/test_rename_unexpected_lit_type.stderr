error: Unexpected literal type `byte string`
 --> $DIR/test_rename_unexpected_lit_type.rs:5:24
  |
5 |     #[shorthand(rename(b"binary"))]
  |                        ^^^^^^^^^
